src/components/filterContainer/FilterContainer.tsx

```diff
@@ -7,11 +7,15 @@ import { FilterType } from './FilterLabel.types'
 
 export const calculateIsSelected = (
   filterType: FilterType,
-  filterName: string,
+  filterName: string | string[],
   selected: string | string[]
 ): boolean => {
   switch (filterType) {
     case 'checkbox':
+      if (Array.isArray(filterName)) {
+        return filterName.some(name => selected.includes(name))
+      }
+
```

Comment:
This will need to be covered with a unit test.

In addition, the ternary syntax is preferred where possible.

Reply:
I think I've added these.